_haikus/i_like_the_cloud.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: I like the cloud
+author: Akliph
+---
+
+I Upload, Download<br>
+Its really helpful to me<br>
+I like the cloud<br>
```

Comment:
There is one syllable missing here! To have the same sentiment, you can add `do`, as in:

```
I do like the cloud<br>
```